src/GitVersionCore/OutputVariables/VersionVariables.cs

```diff
@@ -181,5 +182,32 @@ public bool ContainsKey(string variable)
         private sealed class ReflectionIgnoreAttribute : Attribute
         {
         }
+
+        public override string ToString()
+        {
+            var builder = new StringBuilder();
```

Comment:
Would you prefer I moved this to a JsonBuilder?
So it could be reused in by #2017 ?

Reply:
Hm. How would a generic `JsonBuilder` look like? If you have a suggestion, please add it to the PR.